pkg/proc/threads.go

```diff
@@ -1,585 +1,585 @@
-		// current goroutine is executing on the system stack (sometimes also
-		// referred to as the g0 stack or scheduler stack, I'm not sure if there's
-		// actually any difference between those).
-		// For our purposes it's better if we always return the real goroutine
-		// since the rest of the code assumes the goroutine ID is univocal.
-		// The real 'current goroutine' is stored in g0.m.curg
-		curgvar, err := g.variable.fieldVariable("m").structMember("curg")
-		if err != nil {
-			return nil, err
-		}
-		g, err = curgvar.parseG()
-		if err != nil {
-			return nil, err
-		}
-		g.SystemStack = true
-	}
-	g.Thread = thread
-	if loc, err := thread.Location(); err == nil {
-		g.CurrentLoc = *loc
-	}
-	return g, nil
-}
-
-// ThreadScope returns an EvalScope for this thread.
-func ThreadScope(thread Thread) (*EvalScope, error) {
-	locations, err := ThreadStacktrace(thread, 1)
-	if err != nil {
-		return nil, err
-	}
-	if len(locations) < 1 {
-		return nil, errors.New("could not decode first frame")
-	}
-	return FrameToScope(thread.BinInfo(), thread, nil, locations...), nil
-}
-
-// GoroutineScope returns an EvalScope for the goroutine running on this thread.
-func GoroutineScope(thread Thread) (*EvalScope, error) {
-	locations, err := ThreadStacktrace(thread, 1)
-	if err != nil {
-		return nil, err
-	}
-	if len(locations) < 1 {
-		return nil, errors.New("could not decode first frame")
-	}
-	g, err := GetG(thread)
-	if err != nil {
-		return nil, err
-	}
-	return FrameToScope(thread.BinInfo(), thread, g, locations...), nil
-}
-
-// onNextGoroutine returns true if this thread is on the goroutine requested by the current 'next' command
-func onNextGoroutine(thread Thread, breakpoints *BreakpointMap) (bool, error) {
-	var bp *Breakpoint
-	for i := range breakpoints.M {
-		if breakpoints.M[i].Kind != UserBreakpoint && breakpoints.M[i].internalCond != nil {
-			bp = breakpoints.M[i]
-			break
-		}
-	}
-	if bp == nil {
-		return false, nil
-	}
-	// Internal breakpoint conditions can take multiple different forms:
-	// Step into breakpoints:
-	//   runtime.curg.goid == X
-	// Next or StepOut breakpoints:
-	//   runtime.curg.goid == X && runtime.frameoff == Y
-	// Breakpoints that can be hit either by stepping on a line in the same
-	// function or by returning from the function:
-	//   runtime.curg.goid == X && (runtime.frameoff == Y || runtime.frameoff == Z)
-	// Here we are only interested in testing the runtime.curg.goid clause.
-	w := onNextGoroutineWalker{thread: thread}
-	ast.Walk(&w, bp.internalCond)
-	return w.ret, w.err
-}
-
-type onNextGoroutineWalker struct {
-	thread Thread
-	ret    bool
-	err    error
-}
-
-func (w *onNextGoroutineWalker) Visit(n ast.Node) ast.Visitor {
-	if binx, isbin := n.(*ast.BinaryExpr); isbin && binx.Op == token.EQL && exprToString(binx.X) == "runtime.curg.goid" {
-		w.ret, w.err = evalBreakpointCondition(w.thread, n.(ast.Expr))
-		return nil
-	}
-	return w
-}
+package proc

```

Comment:
You changed the line endings on this file from `\n` (unix) to `\r\n` (windows) this is a big problem because:

1. it makes the diff unreadable going forward
2. I can't review the changes because github marks the entire file as changed.

This needs to be fixed.